### PR TITLE
Add wallet-coin-selection.md to core-dev 2023-09

### PR DESCRIPTION
**Pull request by S3RK**

### CODE REVIEW

**Comment by S3RK** [comment]
@murchandamus @adamjonas could you take a look here?

**Review by murchandamus — APPROVED** [review]

**Comment by adamjonas** [comment]
It's missing the front matter. Mind copying one of these? 

https://github.com/bitcointranscripts/bitcointranscripts/tree/master/bitcoin-core-dev-tech/2023-09

File: bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+* **Topic: review of https://github.com/bitcoin/bitcoin/pull/27601**
```
Comment (murchandamus) [inline bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md:1]:
```suggestion
---
title: Discussion on open Coin Selection matters
categories: ['core-dev-tech']
tags: ['bitcoin core', 'coinselection']
date: 2023-09-19
---

* **Topic: review of https://github.com/bitcoin/bitcoin/pull/27601**
```

Not sure the date is correct. I think it was Tuesday that we talked?

Comment (S3RK) [reply bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md:1]: Yes, it was Tuesday. Accepting the suggestion, will squash later when not on mobile

**Comment by S3RK** [comment]
Squashed and pushed

File: bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
+---
+title: Discussion on open Coin Selection matters
+categories: ['core-dev-tech']
+tags: ['bitcoin core', 'coinselection']
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md:4]:
```suggestion
tags: ['bitcoin core', 'coin selection']
```
`coin selection` already exists as a tag [here](https://github.com/bitcointranscripts/bitcointranscripts/blob/master/bitcoin-core-dev-tech/2023-09/privacy-metrics.md)

File: bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+    * Conclusion so far: Status quo remains
+
+* **Topic: Bug when BnB produce change https://github.com/bitcoin/bitcoin/issues/28180**
+    * Min_viable_change is different from cost_of_change and it doesn't include change_fee
+            * Zeroing out coins selection params for SFFO is bad
+        * We need to fix change_fee
+    * One solution: Reduce upper bound for BnB when SFFO
+    * Another solution: disable BnB and exact knapsack when user requested SFFO
+
+* SFFO use cases are still unclear
```
Comment (kouloumos) [inline bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md:40]: Is this bullet point supposed to be part of the topic above it (Topic: Bug when BnB produce change)?

**Review by kouloumos — COMMENTED** [review]
nit: it seems that the new lines create some sort of unintended spaces.  Preview:
Screenshot from 2023-10-24 17-07-48

I believe that it affects the reading experience but because this kind of behavior has already been merged [as part of another file](https://btctranscripts.com/bitcoin-core-dev-tech/2023-09/kernel-planning/), we could deal with beautification down the line. 
[Here](https://btctranscripts.com/bitcoin-core-dev-tech/2023-09/privacy-metrics/) is how it looks like [in a file](https://github.com/bitcointranscripts/bitcointranscripts/blob/master/bitcoin-core-dev-tech/2023-09/privacy-metrics.md) that doesnt use new lines.

File: bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md (hunk 2 above)
Comment (S3RK) [reply bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md]: Done, thanks

File: bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md (hunk 3 above)
Comment (S3RK) [reply bitcoin-core-dev-tech/2023-09/wallet-coin-selection.md]: Made it part of previous topic.

**Comment by S3RK** [comment]
@kouloumos that's a strange rendering behaviour, thanks for catching. Removed the new lines

**Comment by kouloumos** [comment]
Thank you! and warm welcome as a new contributor! 😎

**Pull request merged**